pkg/apis/kops/componentconfig.go

```diff
@@ -296,6 +296,8 @@ type KubeAPIServerConfig struct {
 	AuditLogMaxSize *int32 `json:"auditLogMaxSize,omitempty" flag:"audit-log-maxsize"`
 	// AuditPolicyFile is the full path to a advanced audit configuration file a.g. /srv/kubernetes/audit.conf
 	AuditPolicyFile string `json:"auditPolicyFile,omitempty" flag:"audit-policy-file"`
+	// AuthenticationTokenWebhook enables bearer token authentication on kubelet.
+	AuthenticationTokenWebhook *bool `json:"authenticationTokenWebhook,omitempty" flag:"authentication-mode-webhook"`
```

Comment:
The flag name should be `authentication-token-webhook`, I believe

Reply:
You're right, I must've blanked. I'll fix it real quick